### ISSUE TITLE
crash when parsing URL

### ISSUE THREAD

**Issue by taviso** [Sambigeara/fuzzynote#94]
If you hit <kbd>Ctrl</kbd>-<kbd>C</kbd> with no URL, `fzn` crashes in `term.(*Terminal).HandleEvent`.

Reproduce:

1. fzn --root /tmp
2. Hit <kbd>Ctrl</kbd>-<kbd>C</kbd>

```
panic: runtime error: invalid memory address or nil pointer dereference
```

**Comment by Sambigeara** [comment]
Sorry, been a bit quiet on this project the last few months, but back to it now 👍

Will address this once another pretty hefty change is merged - thanks for sharing.

**Comment by Sambigeara** [comment]
Hmm, I did just try and recreate and was unable to, though.

I know this was a while ago now (sorry again 🙈), but do you remember if this was consistently occurring? Fresh fzn instances or indeed my own instance aren't raising the same behaviour.

**Comment by taviso** [comment]
Yes, works every time here on Ubuntu focal.

The bug is that `t.c.CurItem` could be nil here:

https://github.com/Sambigeara/fuzzynote/blob/main/pkg/term/client.go#L383

**Comment by Sambigeara** [comment]
Ah, gotcha - I'd been trying to reproduce on items with no URL rather than the top search line. Thanks.

**Comment by Sambigeara** [comment]
Fixed in [v0.24.1](https://github.com/Sambigeara/fuzzynote/releases/tag/v0.24.1)
